src/main/java/com/sourcery/gymapp/backend/workout/model/Workout.java

```diff
@@ -58,4 +58,14 @@ public void setExercises(List<WorkoutExercise> exercises) {
         }
```

Comment:
remove this method (since we are not using in Update scenario @Setter annotation is enough)

Reply:
But once again it's a proper logic of a setter for this kind of field with relationship, the one generated by lombok can cause us some troubles and generate errors.